gnosis/safe/api/transaction_service_api/transaction_service_api.py

```diff
@@ -158,11 +168,13 @@ def get_safe_transaction(
         :param safe_tx_hash:
         :return: SafeTx and `tx-hash` if transaction was executed
         """
-        safe_tx_hash = HexBytes(safe_tx_hash).hex()
-        response = self._get_request(f"/api/v1/multisig-transactions/{safe_tx_hash}/")
+        safe_tx_hash_str = HexBytes(safe_tx_hash).hex()
```

Comment:
No need for this calculation now, right?. Only needed if reponse is not ok

Reply:
It is necessary on the following line to compose the URL

Reply:
You are right, sorry